### DISCUSSION TITLE
How to add an information box, or an html layer

### DISCUSSION THREAD

**Discussion by ninjaKris** [visgl/deck.gl#5897] [Q&A] [answered]
I'm working on a project that has icons on a map. One of the requirements is that each of the icons has an information box (text, icons, etc) placed right above it. It's almost like an always-on tooltip. The problem is that I can't find any layers that do this. Can someone advise?  I need for it to be persistent (ie, not onHover or onClick)

Example:

![mockup](https://user-images.githubusercontent.com/17915305/122817807-3ff25580-d28d-11eb-8378-be8478e5a419.png)

**Comment by Pessimistress** [comment, marked as answer]
You can use a [react-map-gl Marker](http://visgl.github.io/react-map-gl/docs/api-reference/marker).

**Reply by ninjaKris** [reply]
This is what I was looking for. Thank you

**Comment by ibgreen** [comment]
There also some deck.gl compatible [HTML overlays](https://nebula.gl/docs/api-reference/overlays/html-overlay) in https://nebula.gl that may be of interest.

**Comment by ninjaKris** [comment]
@Pessimistress , Markers seemed to solve my issue however I can not click on the href inside my marker box. I looks to be hidden behind the canvas. Playing around with z-index works, but removes the deck layers. Any solution to this?

**Reply by Pessimistress** [reply]
https://deck.gl/docs/api-reference/react/deckgl#contextprovider
